js/data/locale/zh/Hans/HK/unitfmt.json

```diff
@@ -209,29 +223,30 @@
             "square-foot": "#{n}平方英尺",
             "square-inch": "#{n}平方英寸",
             "dunam": "#{n}杜纳亩",
-            "karat": "#{n}克拉",
-            "milligram-per-deciliter": "#{n}毫克/分升",
-            "millimole-per-liter": "#{n}毫摩尔/升",
+            "karat": "#{n} kt",
+            "milligram-ofglucose-per-deciliter": "#{n} mg/dL",
+            "millimole-per-liter": "#{n} mmol/L",
+            "item": "#{n}项",
             "permillion": "#{n}ppm",
             "percent": "#{n}%",
             "permille": "#{n}‰",
             "permyriad": "#{n}‱",
-            "mole": "#{n}摩尔",
-            "liter-per-kilometer": "#{n}升/公里",
-            "liter-per-100-kilometer": "#{n}升/100千米",
-            "mile-per-gallon": "#{n}英里/加仑",
-            "mile-per-gallon-imperial": "#{n}英里/英制加仑",
+            "mole": "#{n} mol",
+            "liter-per-kilometer": "#{n} L/km",
+            "liter-per-100-kilometer": "#{n} L/100km",
+            "mile-per-gallon": "#{n} mpg US",
+            "mile-per-gallon-imperial": "#{n} mpg Imp.",
             "petabyte": "#{n} PB",
-            "terabyte": "#{n}太字节",
-            "terabit": "#{n}太比特",
-            "gigabyte": "#{n}吉字节",
-            "gigabit": "#{n}吉比特",
-            "megabyte": "#{n}兆字节",
-            "megabit": "#{n}兆比特",
-            "kilobyte": "#{n}千字节",
-            "kilobit": "#{n}千比特",
-            "byte": "#{n}字节",
-            "bit": "#{n}比特",
+            "terabyte": "#{n} TB",
```

Comment:
note) Usually iLib prefer to provide their own language translation. but The latest CLDR changed unit translation to english